packages/components/stencil-angularjs-output-target/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@ovh-ux/stencil-angularjs-output-target",
+  "version": "0.0.0",
+  "description": "AngularJS output target for @stencil/core components.",
+  "keywords": [
+    "stencil",
+    "angularjs"
+  ],
+  "homepage": "https://github.com/ovh/manager/tree/master/packages/components/stencil-angularjs-output-target#readme",
+  "bugs": {
+    "url": "https://github.com/ovh/manager/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/components/stencil-angularjs-output-target"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "prebuild": "npx rimraf ./dist",
+    "build": "tsc && npm run rollup",
+    "prepare": "npm run build",
+    "watch": "tsc --watch",
+    "rollup": "rollup -c"
+  },
+  "devDependencies": {
+    "@stencil/core": "^2.3.0",
```

Comment:
Missing diff from `yarn.lock` file then, isn't it?

Reply:
Yes good catch, fixed. Thanks